src/screens/Gurbani/index.tsx

```diff
@@ -27,8 +27,6 @@ const GurbaniScreen = ( {
 }: GurbaniScreenProps ) => {
   const { data } = useQuery( id, loaders[ type ] )
 
-  console.log( `Gurbani Screen: ${type} ${id} ` )
```

Comment:
🤫  😉

Reply:
You can keep these logs they never go out in final builds. Babel will strip them out so no performance overhead

Reply:
I think it just becomes harder for other people to troubleshoot if they're inheriting past console.logs (like way, way in the future)

Reply:
But yeah, I've seen this one -- it's a little helpful too, don't mind either way

Reply:
I followed what eslint had to say! 😅 I don't mind that either!

Reply:
To everyone's points, we do need to add a production-grade logger at some point so that we have the best of both worlds